test-network-function/operator/suite.go

```diff
@@ -70,6 +69,7 @@ var _ = ginkgo.Describe(testSpecName, func() {
 		env := config.GetTestEnvironment()
 		ginkgo.BeforeEach(func() {
 			env.LoadAndRefresh()
+			gomega.Expect(len(env.OperatorsUnderTest)).ToNot(gomega.Equal(0))
```

Comment:
The intent is to fail the test if there is no operator found?

Reply:
yes

Reply:
should we not just skip it ? Is operator deployment mandatory ?

Reply:
exactly, similar to what we did in the scaling and node drain test, skip when no deployments are found.

Reply:
done